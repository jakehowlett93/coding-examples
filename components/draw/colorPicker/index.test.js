import ColorPicker from './index';

describe('color picker', () => {
	const state = {
		room: { roundType: 'draw' }
	};

	const actions = {
		onColorChange: jest.fn()
	};
	let returnValue;
	beforeEach(() => {
		returnValue = ColorPicker({ state, actions });
	});
	it('should write the right component ', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
