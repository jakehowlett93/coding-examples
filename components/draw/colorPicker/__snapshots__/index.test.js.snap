// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color picker should write the right component  1`] = `
<ColorPicker
  circleSize={28}
  circleSpacing={14}
  color={
    Object {
      "a": 1,
      "h": 250,
      "l": 0.2,
      "s": 0.5,
    }
  }
  colors={
    Array [
      "#f44336",
      "#e91e63",
      "#9c27b0",
      "#673ab7",
      "#3f51b5",
      "#2196f3",
      "#03a9f4",
      "#fff",
      "#009688",
      "#4caf50",
      "#8bc34a",
      "#cddc39",
      "#ffeb3b",
      "#ffc107",
      "#ff9800",
      "#000",
      "#795548",
      "#607d8b",
    ]
  }
  styles={Object {}}
  width={252}
/>
`;
